src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -291,4 +292,34 @@ private List<Error> validatePackagePoLine(CompositePoLine compPOL) {
 
     return convertErrorCodesToErrors(compPOL, errors);
   }
+
+  public List<Error> validateForBinadryActive(CompositePoLine poLine) {
+    List<Error> errors = new ArrayList<>();
+    if (poLine.getDetails() != null && poLine.getDetails().getIsBindaryActive()) {
```

Comment:
I saw that isBindaryActive() false by default, what it be for old orders created before adding this flag?
If it null - we will have NPE here

Reply:
In case of null, Used Boolean.TRUE.equals